pennylane/devices/default_qubit_jax.py

```diff
@@ -145,6 +145,9 @@ def circuit():
         "CRY": jax_ops.CRY,
         "CRZ": jax_ops.CRZ,
         "MultiRZ": jax_ops.MultiRZ,
+        "DoubleExcitation": jax_ops.DoubleExcitation,
+        "DoubleExcitationPlus": jax_ops.DoubleExcitationPlus,
+        "DoubleExcitationMinus": jax_ops.DoubleExcitationMinus,
     }
 
     C_DTYPE = jnp.complex64

```

Comment:
just out of curiosity, why is the precision lowered here as compare with autograd and tensorflow?

Reply:
@Thenerdstation